video/image_writer.c

```diff
@@ -15,15 +15,30 @@
  * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
 #include <libavcodec/avcodec.h>
 #include <libavutil/mem.h>
 #include <libavutil/opt.h>
+#include <libavutil/pixdesc.h>
 
+#include "common/msg.h"
 #include "config.h"
+#include "libavcodec/codec.h"
+#include "libavcodec/codec_id.h"
+#include "libavcodec/packet.h"
+#include "libavformat/avformat.h"
+#include "libavformat/avio.h"
+#include "libavutil/avutil.h"
+#include "libavutil/dict.h"
+#include "libavutil/frame.h"
+#include "libavutil/pixfmt.h"
+#include "libavutil/rational.h"
+#include "osdep/timer.h"
+#include "video/csputils.h"
```

Comment:
From this big list of includes you added, as far as I can tell you only need `libavformat/avformat.h` and `libavutil/pixdesc.h`. By the way, the libav stuff should use brackets, i.e. `#include <libavformat/avformat.h>`

Reply:
done.